functions.php

```diff
@@ -93,17 +94,27 @@ function prefillEditData(array $singleData): string
 {
     foreach($singleData as $index) 
         {
-        $data = '<div class="winners-box"><form action="edit.php" method="POST"><label for="editSeason">What year was the final played?</label><br><input type="text" id="season" name="editSeason" value="'.$index['Season'].'"/><br><br>' . '<label for="editPremier">Who won the flag?</label><br>' . '<input type="text" id="premier" name="editPremier" value="'.$index['Premier'].'"/><br><br>' . '<label for="editRunnersUp">Who did they play?</label><br>' . '<input type="text" id="runnersUp" name="editRunnersUp" value="'.$index['Runner-Up'].'"/><br><br>' . '<label for="editScore">What was the score? (GG.PP (TOTAL) d. GG.PP (TOTAL))</label><br>' . '<input type="text" id="score" name="editScore" value="'.$index['Score'].'"/><br><br>' . '<input class="hidden" name="dataID" value="' . $index['id'] . '">' . '<input type="submit"/>' . '</form></div>';
-        
+        $data = '<div class="winners-box"><form class="editForms" action="hiddenEdit.php" method="POST"><label for="editSeason">What year was the final played?</label><br><input type="text" id="season" name="editSeason" value="'.$index['Season'].'"/><br><br>' . '<label for="editPremier">Who won the flag?</label><br>' . '<input type="text" id="premier" name="editPremier" value="'.$index['Premier'].'"/><br><br>' . '<label for="editRunnersUp">Who did they play?</label><br>' . '<input type="text" id="runnersUp" name="editRunnersUp" value="'.$index['Runner-Up'].'"/><br><br>' . '<label for="editScore">What was the score? (GG.PP (TOTAL) d. GG.PP (TOTAL))</label><br>' . '<input type="text" id="score" name="editScore" value="'.$index['Score'].'"/><br><br>' . '<input class="hidden" name="dataID" value="' . $index['id'] . '">' . '<input type="submit"/>' . '</form></div>';
         }    
     return $data;
 }
 
-
+/**
+ * Connects edits to the database to update the index page
+ *
+ * @param [type] $db
+ * @param [type] $editSeason
+ * @param [type] $editPremier
+ * @param [type] $editRunnersUp
+ * @param [type] $editScore
+ * @param [type] $id
+ * @return void
+ */
 function editData($db, $editSeason, $editPremier, $editRunnersUp, $editScore, $id) 
```

Comment:
missing type hints and return type

Reply:
type hints and return type updated

Reply:
good for the docblock but the actual function needs these typehints on them too:
```php
function editData(PDO $db, string $editSeason, string $editPremier, string $editRunnersUp, int $editScore, int $id): bool 
```